content/hi/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,125 @@
+---
+title: सीग्रुप v2 के बारे में
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+Linux पर, {{< glossary_tooltip text="control groups" term_id="cgroup" >}}
+प्रक्रियाओं को आवंटित संसाधनों को सीमित करें।
```

Comment:
instead of `प्रक्रियाओं को आवंटित संसाधनों को सीमित करें।`   I hope प्रक्रियाओं को आवंटित संसाधनों को सीमित करते हैं।  will looks good